app/cypress/e2e/configure-output.cy.ts

```diff
@@ -0,0 +1,58 @@
+describe("Configure output tests", () => {
+    beforeEach(() => {
+        sessionStorage.clear();
+    });
+
+    const planName = 'Test Plan';
+    const planDescription = 'test description';
+
+    it("Verify Name and Description", () => {
+        cy.mapLoadDelay();
+        cy.findByRole('button', { name: 'Create Plan' }).should('exist').click();
+
+        cy.findByPlaceholderText("Enter Plan Name").type(planName);
+        cy.get('#scenario-description-input').type(planDescription);
+
+        cy.findByRole('button', { name: 'Save' }).click();
+
+        cy.findByRole('button', { name: 'Configure Output' }).should('exist').click();
+        cy.contains(planName).should('exist');
+        cy.contains(planDescription).should('exist');
+    });
+
+    it("Verify texts", () => {
+        cy.mapLoadDelay();
+        cy.findByRole('button', { name: 'Create Plan' }).should('exist').click();
+
+        cy.findByPlaceholderText("Enter Plan Name").type(planName);
+        cy.get('#scenario-description-input').type(planDescription);
+
+        cy.findByRole('button', { name: 'Save' }).click();
+
+        cy.findByRole('button', { name: 'Configure Output' }).click();
+        cy.get('h2').contains('Configure Output').should('exist');
```

Comment:
```suggestion
        cy.findByRole("heading", { name: "Configure Output" });
```

Here is a way to verify you are on the `Configure Output` tab without using `cy.get`. This also has the benifit of verifying accessibility.